locales/ja-JP.json

```diff
@@ -0,0 +1,7 @@
+{
+  "talk_to": "につないで",
```

Comment:
Overall, I do like this change as it is much cleaner than before. I do wonder, in terms of scale, how this may work. Should this library have localization for every language, or should we have some sort of documented warning if you set the locale to an unsupported version?

As the platform adds new languages, should this library be responsible for adding translations for each language at the same time?

Reply:
@Fleker Thank you for the reviewing!

> Should this library have localization for every language, or should we have some sort of documented warning if you set the locale to an unsupported version?

Good point. Well, I guess that we should warn if users set the unsupported locale using `setLocale(...)`. I forgot the implementation. I intend to add the code to do that.

> As the platform adds new languages, should this library be responsible for adding translations for each language at the same time?

Yes, I think we need to add a message bundle file at the same time when a new language is added to the platform. Because, each grammar of languages is different.

Invocation phrase for each language are defined on the following page:
https://developers.google.com/actions/localization/languages-locales

Two grammars of English and French are almost same. The location of the app name is last.

* English: "talk to $name"
* French: "parler avec $prénom"

However, grammars of German, Japanese and some languages are different from them:

* German: "mit $name sprechen" (The location of the app name is middle)
* Japanese: "$nameにつないで" (The location of the app name is first)

Currently, we prepare definitions for three languages. But, I intend to create all languages which are defined on the page above. I will commit the files later.

Also, it seems that the current definition of each `start_conversation` and `start_conversation_with_prompt` is complex.

* Ex: `"start_conversation": "%1$s %2$s"`

I will change the definition to `"start_conversation": "talk to %s"` to become more clearly.